composer.json

```diff
@@ -10,10 +10,10 @@
         "psr-4": {"Lullabot\\Drainpipe\\Tests\\Functional\\": "tests/src/functional/"}
     },
     "require": {
-        "php": "^8.1",
+        "php": "^8.2",
```

Comment:
Looks like PHP 8.1 is still supported until Jan. 1 2026.
* https://www.php.net/supported-versions.php

Wouldn't `^8.1`  still allow PHP 8.2 to be installed?
Why do we need to force a minimum of PHP 8.2?